### ISSUE TITLE
Form doesn't load next page when creating datasets

### ISSUE THREAD

**Issue by Floppy** [theodi/octopub#594]
The create dataset form doesn't go to the next page when uploading. The app returns a 202 code. Datasets are created properly, but the user just sees a spinning submit button.

Seen on FSA staging instance, which is the same code as octopub.io, but I've not verified it on octopub.io yet.

**Comment by Floppy** [comment]
Log:

```
2017-09-04T23:32:00.703818+00:00 app[web.1]: [f79290a1-1b9e-4df2-a10d-4eef800d72cc]   Parameters: {"utf8"=>"✓", "authenticity_token"=>"qyGfCAqXGRqUvYiUTbcCmWCnDG7Qlm/hDxKb8ZX7qkdYJC1m+rhkMQnE4makos+KBBGYyKOBwn0mh7b13LGa5A==", "dataset"=>{"name"=>"testing again", "owner"=>"floppy", "description"=>"", "publisher_name"=>"Publisher name change", "publisher_url"=>"http://floppy.org.uk", "license"=>"CC-BY-4.0", "publishing_method"=>"github_public", "frequency"=>"One-off"}, "files"=>[{"title"=>"fdsfdsfds", "description"=>"", "file"=>"uploads/3cf00624-d722-491b-ac9f-6e3364296da3/aggregation-by-country.csv", "dataset_file_schema_id"=>"", "schema_name"=>"", "schema_description"=>"", "schema_restricted"=>"true"}], "async"=>"true", "channel_id"=>"post-05030302-0404-4305-8b0e-0008020f0d06"}
2017-09-04T23:32:00.704236+00:00 app[web.1]: [f79290a1-1b9e-4df2-a10d-4eef800d72cc] DatasetsController: In process_files
2017-09-04T23:32:00.704301+00:00 app[web.1]: [f79290a1-1b9e-4df2-a10d-4eef800d72cc] file is not an http uploaded file, it's an s3 storage key
2017-09-04T23:32:00.704383+00:00 app[web.1]: [f79290a1-1b9e-4df2-a10d-4eef800d72cc] DatasetsController: In check_mandatory_fields
2017-09-04T23:32:01.030056+00:00 app[web.1]: [f79290a1-1b9e-4df2-a10d-4eef800d72cc] DatasetsController: In create
2017-09-04T23:32:01.042029+00:00 app[web.1]: [f79290a1-1b9e-4df2-a10d-4eef800d72cc] DatasetsController: In create with params aysnc
2017-09-04T23:32:01.042329+00:00 app[web.1]: [f79290a1-1b9e-4df2-a10d-4eef800d72cc] Completed 202 Accepted in 338ms (ActiveRecord: 1.1ms)
```

**Comment by Floppy** [comment]
Replicated locally. Server is returning a 202 when the upload is complete.

**Comment by Floppy** [comment]
This now appears to be OK on octopub.io. Perhaps related to the whitespace fix that was reverted in #603?

**Comment by Floppy** [comment]
verified on fsa-octopub as well. Strange.
